toolkit/tools/internal/safechroot/safechroot.go

```diff
@@ -473,21 +478,22 @@ func cleanupAllChroots() {
 // the chroot until the unmounts succeed or too many failed attempts.
 // This is to avoid leaving folders like /dev mounted when the chroot folder is forcefully deleted in cleanup.
 // Iff all mounts were successfully unmounted, the chroot's root directory will be removed if requested.
-func (c *Chroot) unmountAndRemove(leaveOnDisk, isFinalAttempt bool) (err error) {
+// If doLazyUnmount is true, use the lazy unmount flag which will allow the unmount to succeed even if the mount point is busy.
+func (c *Chroot) unmountAndRemove(leaveOnDisk, doLazyUnmount bool) (err error) {
```

Comment:
`doLazyUnmount` is both a function parameter and a constant. Please avoid name hiding.

Reply:
Nice catch!